### ISSUE TITLE
IntelliJ produces Diamond-Inspection warning when specifying generic type on val declaration

### ISSUE THREAD

**Issue by depsypher** [mplushnikov/lombok-intellij-plugin#153]
Would love to be able to write code like this...
```val example = new ArrayList<String>();```

without getting warnings like this:
```Explicit type argument String can be replaced with <>```

**Comment by nallar** [comment]
Issue is still present in 0.9.7.

![IDEA 15.0.2 with Lombok Plugin 0.9.7](https://i.imgur.com/q2c4Lkw.png)

sort-of-workaround is to use `@SuppressWarnings("Convert2Diamond")` or disable the inspection globally.

**Comment by mplushnikov** [comment]
Just added some workarounds to skip Diamond-inspection on val declarations. You will be able to verify it in the next release.

**Comment by mplushnikov** [comment]
Plugin 0.9.8 was released today

**Comment by kirviq** [comment]
Hi

I configured Diamond-Inspections to show up as notices (because there is nothing really wrong with the code, it just could be a little nicer) and all of a sudden my setting was changed back to warning again.
Some researching and experimenting (turning the lombok plugin off) later I came across this issue and found out that it's enough to turn off the one line in the config file that overrides the IntelliJ inspection.

Personally I would much prefer living with a notification I can't do anything about when I use a val than with thousands of warnings where nothing is actually broken. (Yes I'm dealing with a large and partially old code base.)

Would it be too much to ask for an option to turn this inspection-overriding off?
